### PR TITLE
Referral program rewards admin reporting

### PR DESCRIPTION
**Pull request by kgorazd**

### CODE REVIEW

File: lib/tasks/report.rake [hunk 1]
```diff
@@ -0,0 +1,17 @@
+namespace :referral_program do
+  desc 'Sends daily referral program rewards report'
```
Comment (dimon-k) [inline lib/tasks/report.rake:2]: "Send's... " :)

File: app/mailers/spree/referral_program_mailer.rb [hunk 2]
```diff
@@ -1,18 +1,32 @@
...
+      return if admin_emails.empty?
+
+      subject = "[#{Spree::Store.current.name}] #{Spree.t('referral_program_mailer.report_email.subject')} - #{scope_name}"
+      mail(to: admin_emails, from: from_address, subject: subject)
+    end
+
+    private
+
+    def admin_emails
+      admin_emails = ENV['REFERRAL_PROGRAM_ADMIN_EMAILS'].to_s.split(',').map(&:strip)
```
Comment (dimon-k) [inline app/mailers/spree/referral_program_mailer.rb]: Should we use `@admin_emails ||= ENV...` ?

**Review by dimon-k — APPROVED** [review]

File: app/mailers/spree/referral_program_mailer.rb (hunk 2 above)
Comment (kgorazd) [reply app/mailers/spree/referral_program_mailer.rb]: Fixed

File: lib/tasks/report.rake (hunk 1 above)
Comment (kgorazd) [reply lib/tasks/report.rake:2]: Are you sure? https://www.wordwebonline.com/en/SEND, http://www.verbix.com/webverbix/English/send.html

**Pull request merged**